src/tests/sqlsmith/src/bin/main.rs

```diff
@@ -89,7 +89,6 @@ async fn create_tables(
 
     let mut setup_sql = String::with_capacity(1000);
     let sql = get_seed_table_sql(opt);
-    setup_sql.push_str(&sql);
```

Comment:
Already done below: https://github.com/singularity-data/risingwave/pull/4806/files#diff-7e6fe866c69eb80b36e31cb74ecb2995b6f820dfc5ccddfe152600cfbbacad62R100

Reply:
Hence we can remove it.